README.md

```diff
@@ -45,120 +61,103 @@ flowchart LR;
+|---------|-------------|-------------------|--------|
+| Account Service | Manages user accounts and authentication on the Frequency blockchain. It handles tasks such as account creation and key management. | [API Docs](https://projectlibertylabs.github.io/gateway/account) | [README](./developer-docs/account/README.md) |
+| Graph Service | Manages social connections and relationships between users on the Frequency network. It handles operations like following/unfollowing users and retrieving social graphs. | [API Docs](https://projectlibertylabs.github.io/gateway/graph/) | [README](./developer-docs/graph/README.md) |
+| Content Publishing Service | Facilitates the creation and publication of content on the Frequency blockchain. It manages tasks such as posting messages, creating threads, and content moderation. | [API Docs](https://projectlibertylabs.github.io/gateway/content-publishing/) | [README](./developer-docs/content-publishing/README.md) |
+| Content Watcher Service | Monitors and retrieves content updates from the Frequency blockchain. It allows applications to efficiently track and display new content as it's published. | [API Docs](https://projectlibertylabs.github.io/gateway/content-watcher/) | [README](./developer-docs/content-watcher/README.md) |
 
-<details>
-<summary>Content Watcher Service</summary>
-
-- [API Documentation](https://projectlibertylabs.github.io/gateway/content-watcher/)
-- [README](./developer-docs/content-watcher/README.md)
-
-</details>
-
-<!-- LIVE Docs -->
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
 
-## 🚀 Live Docs
+### Key Features of Gateway Microservices
 
-- [Live Docs](https://projectlibertylabs.github.io/gateway/)
+- **Modularity**: Each service can be used independently, allowing developers to integrate only the functionalities they need.
+- **Scalability**: The microservice architecture allows for easy scaling of individual services based on demand.
+- **Simplified Blockchain Interaction**: These services abstract complex blockchain operations, providing a familiar API-based interface for developers.
+- **Comprehensive Documentation**: Each service comes with detailed API documentation and a README file for easy integration and usage.
 
 <p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
 
-## 💻 Getting Started
-
-For a more detailed tutorial, visit [Live Docs](https://projectlibertylabs.github.io/gateway/).
-
-To get a local copy up and running, follow these steps.
+<!-- GETTING STARTED -->
 
-### Prerequisites
+## 💻 Getting Started <a name="getting-started"></a>
 
-In order to run this project you need:
+Gateway offers flexibility in how you can use its services. You can either quickly set up all services or selectively use specific microservices based on your needs. Below, we provide both a Quick Start guide for setting up all services and information on how to get started with individual microservices.
```

Comment:
can we link to the quick start?

Reply:
Yes, we have a link in the table of contents the content is right below this sentence.